pages/_staticPage.vue

```diff
@@ -5,21 +5,48 @@
     <picture v-for="image in images" v-bind:key="image.id">
       <img v-bind:src="`${image.imgixHost}${image.value.path}?w=640&q=65`"/>
     </picture>
-    <section>
-      {{body}}
+    <section v-html="htmlBody">
     </section>
   </article>
 </template>
 
 <script>
 import loadData from '~/lib/load-data'
+import marked from 'marked'
+
+const renderer = new marked.Renderer()
```

Comment:
Because this is the second time we use this part of code it would be nice to move it to a lib function and include it.